crates/astria-core-crypto/src/lib.rs

```diff
@@ -13,6 +13,7 @@ use std::{
     sync::OnceLock,
 };
 
+use astria_core_consts::ADDRESS_LENGTH;
```

Comment:
This should be reexported because its part of the crate's public interface (via `SigningKey::address_bytes -> [u8; ADDRESS_LENGTH]` and `VerificationKey::address_bytes -> [u8; ADDRESS_LENGTH]`)

Reply:
Done in [d22d29f](https://github.com/astriaorg/astria/pull/1800/commits/d22d29fa87208bf4c24719eb1d7a18ff1b1b84cd)